content/docs/rendering-elements.md

```diff
@@ -52,7 +52,7 @@ Consider this ticking clock example:
 
 `embed:rendering-elements/update-rendered-element.js`
 
-[](codepen://rendering-elements/update-rendered-element)
+**[Try it on CodePen](https://codepen.io/aniruddha-shriwant/pen/WNZVRzM?editors=1011)**
```

Comment:
https://codepen.io/gaearon/pen/gwoJZk ? This one?

Reply:
💯